src/simulator_core/entities/assets/production_cluster.py

```diff
@@ -290,14 +284,14 @@ def write_to_output(self) -> None:
         # Retrieve the general model setpoints (Ts, Tr, Qh)
         setpoints = self.get_setpoints()
         # Retrieve the mass flow (mdot)
-        setpoints[PROPERTY_MASSFLOW] = self.pandapipes_net.res_circ_pump_mass["mdot_flow_kg_per_s"][
+        setpoints[PROPERTY_MASSFLOW] = self.pandapipes_net.circ_pump_mass["mdot_flow_kg_per_s"][
```

Comment:
This should really be res_junction otherwise it won't work. The "pn_bar" properties are not available for just the junctions. You'll need the results!

Reply:
Adjusted